app/assets/stylesheets/govuk_publishing_components/components/_action-link.scss

```diff
@@ -63,6 +63,20 @@
   }
 }
 
+.gem-c-action-link--mobile-subtext {
+  .gem-c-action-link__subtext-wrapper {
+    display: block;
+  }
+
+  .gem-c-action-link__subtext {
+    padding: 0;
+
+    &:before {
+      display: none;
+    }
```

Comment:
This looks strange. Is it putting a nothing before the item?

Reply:
It's hiding the vertical line that appears when the link and subtext are laid out in a row, which is achieved using this `before` element:

<img width="395" alt="Screenshot 2020-06-02 at 12 34 26" src="https://user-images.githubusercontent.com/861310/83515522-6c460300-a4cd-11ea-952d-e5466a9de266.png">

Reply:
Thank you